components/create-bounty-modal.tsx

```diff
@@ -306,26 +288,26 @@ export default function CreateBountyModal() {
           {rewardChecked && renderBountyToken(true, "reward")}
           <div className="container">
             <div className="row">
-              <div className="col-md-6">
+              <div className="col-md-6 text-truncate">
                 <label className="caption-small mb-2">
                   {t("bounty:review.repository")}
                 </label>
-                <GithubInfo
-                  parent="list"
-                  variant="repository"
-                  label={repository?.path}
-                  simpleDisabled={true}
-                />
+              <GithubInfo
+                parent="list"
+                variant="repository"
+                label={repository?.path.length <= 20 ? repository?.path : repository?.path.slice(0,20)+"..."}
```

Comment:
this should be done via css

Reply:
in this situation the component cannot have the dimension to perform the truncate. I put the truncate in the div above, but it didn't have such a nice visual effect. So I put this slice to improve the viewing experience